packages/app-builder-lib/src/targets/FpmTarget.ts

```diff
@@ -196,6 +196,22 @@ export default class FpmTarget extends Target {
       }
     }
 
+    if (target === "deb") {
+      const recommends = (options as DebOptions).recommends
+      if (recommends != null) {
+        if (Array.isArray(recommends)) {
+          fpmConfiguration.customRecommends = recommends
+        } else {
+          // noinspection SuspiciousTypeOfGuard
+          if (typeof recommends === "string") {
+            fpmConfiguration.customRecommends = [recommends as string]
+          } else {
+            throw new Error(`recommends must be Array or String, but specified as: ${recommends}`)
+          }
+        }
+      }
+    }
+
```

Comment:
Since the schema validator will automatically flag incorrect types being passed in, we can simplify this to use `asArray`. I think we should also check a general falsey for `if (recommends) {...}`, as I don't think we should allow an empty string to be passed into customRecommends. What are your thoughts?
```
import { asArray } from "builder-util"

if (target === "deb") {
  const recommends = (options as DebOptions).recommends
  if (recommends) {
      fpmConfiguration.customRecommends = asArray(recommends)
  }
}
```